static/src/javascripts/projects/common/modules/experiments/tests/commercial-us-mobile-sticky.js

```diff
@@ -8,8 +8,8 @@ export const commercialUsMobileSticky: ABTest = {
     expiry: '2019-09-30',
     author: 'Ricardo Costa',
     description: 'This test runs the new US mobile sticky ad slot',
-    audience: 0.0,
-    audienceOffset: 0.0,
+    audience: 0.2,
+    audienceOffset: 0.8, // Offsetting this just because I'm a rebel
```

Comment:
🙃Actually a good idea to offset if there are other active tests! 

You can check participations here: https://frontend.gutools.co.uk/analytics/abtests

Reply:
I'm a data conscious rebel! The idea behind this offset actually is that since this is a very sensitive AB test I considered it prudent to set it apart preemptively. Specially since we've been introducing a few AB tests around the consent banner which uses the same area of the viewport.